obsplus/events/get_events.py

```diff
@@ -11,23 +11,68 @@
 
 import obsplus
 
-UNSUPPORTED_PARAMS = {
-    "minradius",
-    "maxradius",
-    "latitude",
-    "longitude",
-    "magnitude_type",
-    "events",
-    "contributor",
-}
+CIRCULAR_PARAMS = {"latitude", "longitude", "minradius", "maxradius"}
+
+NONCIRCULAR_PARAMS = {"minlongitude", "maxlongitude", "minlatitude", "maxlatitude"}
+
+UNSUPPORTED_PARAMS = {"magnitude_type", "events", "contributor"}
 CLIENT_SUPPORTED = set(inspect.signature(Client.get_events).parameters)
 SUPPORTED_PARAMS = CLIENT_SUPPORTED - UNSUPPORTED_PARAMS
 
 
+def _sanitize_circular_search(**kwargs):
```

Comment:
Do you mind just adding a small docstring (one line) to this to explain what it does?